doc/ref_model/chapters/chapter05.md

```diff
@@ -223,7 +223,7 @@ The host profile and capabilities include:
 1. **Local Disk Capacity**: is the # of local disks and teh capacity of the disks installed on the physical server.
 1. **SMT/HT (SMT: Simultaneous Multithreading/ HT: Hyper Threading)**: Enabled on all physical servers. Gets multiple threads per physical core. Always ON. Configured in the host.
 1. **NUMA (Non-Uniform Memory Access)**: Indicates that vCPU will be on a Socket that is aligned with the associated NIC card and memory. Important for performance optimized VNFs. Configured in the host.
-1. **SR-IOV (Single-Root Input/Output Virtualisation)**: Configure PCIe ports to support SR-IOV.
+1. **SR-IOV (Single-Root Input/Output Virtualisation)**: Configure PCIe ports to support SR-IOV (without exposing SR-IOV VFs to workload via PCI-Pass through).
```

Comment:
Maybe change in wording: Configure multiple Virtual Functions (VF) on the PCIe ports on SR-IOV enabled cards.

Reply:
In addition to this, we should define `SR-IOV` in glossary. We can mention there that PCI-passthrough is not part of it.

Reply:
@pgoyal01 the NFVI doesn't configure VF ports, it enables SRIOV on SRIOV devices.

Reply:
@deebhatia if it has to be used on different place in RM, we can move it to glossary. I think it should be OK.